src/NuGetForUnity/Editor/NugetWindow.cs

```diff
@@ -836,41 +932,50 @@ private void DrawPackage(NugetPackage package, GUIStyle packageStyle, GUIStyle c
 
                     EditorStyles.label.fontSize = 10;
                     EditorStyles.label.fontStyle = FontStyle.Normal;
-                    rect.y += EditorStyles.label.fontSize / 2;
+                    rect.y += EditorStyles.label.fontSize / 2f;
 
                     if (!string.IsNullOrEmpty(package.Authors))
                     {
-                        var authorLabel = string.Format("by {0}", package.Authors);
+                        var authorLabel = $"by {package.Authors}";
                         var size = EditorStyles.label.CalcSize(new GUIContent(authorLabel));
                         GUI.Label(rect, authorLabel, EditorStyles.label);
                         rect.x += size.x + paddingX;
                     }
 
                     if (package.DownloadCount > 0)
                     {
-                        var downloadLabel = string.Format("{0} downloads", package.DownloadCount.ToString("#,#"));
-                        var size = EditorStyles.label.CalcSize(new GUIContent(downloadLabel));
+                        var downloadLabel = $"{package.DownloadCount:#,#} downloads";
                         GUI.Label(rect, downloadLabel, EditorStyles.label);
                     }
                 }
 
                 GUILayout.FlexibleSpace();
                 if (installed != null && installed.Version != package.Version)
                 {
-                    GUILayout.Label(string.Format("Current Version {0}", installed.Version));
+                    GUILayout.Label($"Current Version {installed.Version}");
                 }
 
-                GUILayout.Label(string.Format("Version {0}", package.Version));
+                GUILayout.Label($"Version {package.Version}");
 
-                if (installedPackages.Contains(package))
+                if (installed != null && installed.Version == package.Version)
```

Comment:
I added this check because 'contains' only compares `Id` and `Version` but not `IsManuallyInstalled`, so there was a bug when previewing installed packages in 'Online' tab where all packages were marked as not manually installed. Also this if block was meant to work with already installed packages as I understood it, so it made more sense to use 'installed' variable instead of 'package' which can be passed from Online tab and therefore from 'availablePackages' array. So the version check is there just to replace the 'Contains' condition.

Reply:
It seems both you @JoC0de and us are missing something :). So the original code showed Uninstall button if exact package version was installed, but if a different version was installed there would be no Uninstall button and only Upgrade/Downgrade button. We leaned on this logic and wrongly also added "Add as explicit" button only if exact version is installed.

If you agree we would change it so that:
1. Uninstall button is visible if any version of the package is installed.
2. Downgrade/Upgrade button is only visible if different package version is installed, same as now.
3. Add as explicit button is visible if any version of the package is installed but not manually.

Reply:
@igor84 yes I agree. I think it was never a issue beaus in the "Installed" tab the version is always the same so the Uninstall was only, wrongly, hidden if you find a package that is already installed and there is a update.